docs/nix-getting-started.md

```diff
@@ -93,50 +54,7 @@ Add `.direnv` to your `.gitignore`.
 nix-shell '<home-manager>' -A install
 ```
 
-Installing home manager should create a `.config/home-manager/home.nix` that looks something like this:
-
-```nix
-{ config, pkgs, lib, ... }:
-{
-  # Let Home Manager install and manage itself.
-  programs.home-manager.enable = true;
-
-  # Home Manager needs a bit of information about you and the
-  # paths it should manage.
-  home.username = "<your name>";
-  home.homeDirectory = "/home/<your name>";
-  xdg.cacheHome = "/scratch/<your name>/.cache";
-  xdg.stateHome = "/scratch/<your name>/.local/share";
```

Comment:
I don't think we have those in our template. But would be good to have, I guess.

Reply:
I'll create a follow-up PR adding those.